### ISSUE TITLE
{Linked:Bug2612115}[Android][Accessibility] Keyboard focus is going to non-interactive text using tab/shift +tab key.

### ISSUE THREAD

**Issue by nbystndr** [microsoft/AdaptiveCards#7959]
### Target Platforms

Android

### SDK Version

2.8.2

### Application Name

Microsoft Teams

#### Actual Results: 
Keyboard focus is going to non-interactive text using tab/shift +tab key.

#### Expected Result:
Keyboard focus should move only interactive control using tab/shift +tab.

#### User Experience:
Users who rely on keyboard will try to activate the controls if keyboard focus is going on any non-interactive control using tab/shift.

#### MAS Reference:
[Focus Order - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/02.04.03)

### Screenshots

https://user-images.githubusercontent.com/9493961/194744732-afc70218-eb3e-4e83-94ea-8a2ba98623fa.mp4

### Card JSON

```JSON
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.4",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "width": 2,
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Tell us about yourself",
                            "weight": "Bolder",
                            "size": "Medium"
                        },
                        {
                            "type": "TextBlock",
                            "text": "We just need a few more details to get you booked for the trip of a lifetime!",
                            "isSubtle": true,
                            "wrap": true
                        },
                        {
                            "type": "TextBlock",
                            "text": "Your name",
                            "wrap": true
                        },
                        {
                            "type": "Input.Text",
                            "id": "myName",
                            "placeholder": "Last, First"
                        }
                    ]
                },
                {
                    "type": "Column",
                    "width": 1,
                    "items": [
                        {
                            "type": "Image",
                            "url": "https://upload.wikimedia.org/wikipedia/commons/b/b2/Diver_Silhouette%2C_Great_Barrier_Reef.jpg",
                            "altText": "Diver in the Great Barrier Reef"
                        }
                    ]
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Submit"
        }
    ]
}
```

**Comment by longlong-cs** [comment]
@paulcam206 this will downgrade our feature's accessibility grading, could you help take a look and provide an ETA? thanks!

**Comment by anna-dingler** [comment]
@nbystndr Do you have any additional information about your device's settings?

If a user needs TalkBack **and** keyboard navigation, we want focus to land on TextBlocks so they are read by TalkBack.

In the provided recording, do you have TalkBack enabled? If not, do you have any other accessibility tools running?

**Comment by nbystndr** [comment]
@anna-dingler Turn off the Talkback, just connect your device to a Bluetooth keyboard or test it in Android simulator and use keyboard to navigate. Use tab/shift + tab to move keyboard focus.

**Comment by Raisul123** [comment]
Issue is still repro on below Test Environment:
Android Version: 13
App: Adaptive Cards Android Visualizer
App Version: 2022.12.01.1 (60498133)
Tool: Logitech Bluetooth Keyboard

PFA for reference
https://user-images.githubusercontent.com/23114460/205847856-792108c3-5dba-4d20-95f2-d8cf63cfd40a.mp4

**Comment by longlong-cs** [comment]
@Raisul123 cannot repro for me with the apk I build locally with latest main branch, maybe you can check whether you install and turn on the Accessibility Insights for Android service, seems it may affect the tab focus, if you have try to turn if off and verify again.

**Comment by anna-dingler** [comment]
@nbystndr @Raisul123 Can you confirm that you can repro this issue with Accessibility Insights for Android _disabled_?

Some accessibility services (ex: Talkback) require that we focus on non-interactive elements. Accessibility Insights for Android falsely triggers this logic because it registers the same `AccessibilityServiceInfo`.

**Comment by anna-dingler** [comment]
Hi @Raisul123,

I am closing this issue because I am unable to reproduce the error. I believe the issue you are seeing is caused by Accessibility Insights for Android.

Let me know if you have any questions - thanks!
